css/lemuel/styles.css

```diff
@@ -0,0 +1,19 @@
+* {
+    border-radius: 0 !important;
```

Comment:
Is !important really necessary? Why don't you want anything to override this?
What does !important do?